CHANGELOG.md

```diff
@@ -632,6 +637,12 @@ Other minor changes
                     CommutativeRing (a ⊔ b) (ℓ₁ ⊔ ℓ₂)
  ```
 
+* Added new functions to `Algebra.Definitions.RawSemiring`:
+  ```agda
+  _^[_]*_ : A → ℕ → A → A
+  _^ᵗ_     : A → ℕ → A
+  ```
```

Comment:
I think we're missing entries for the addition of `_^_` to `Data.Integer.Base` and the proofs to `Data.Integer.Properties`?